aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -683,6 +687,17 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
 #undef BINARY_POINTWISE_VA
 #undef BINARY_POINTWISE
 
+
+#define TRIVIAL_OP(op) m.impl(#op, \
+    unwrap_and_call<Tensor (*)(const Tensor&), at::op>);
+  // complex number view operators
+  TRIVIAL_OP(imag)
+  TRIVIAL_OP(conj);
```

Comment:
conj is not a view for complex Tensor?

Reply:
Thanks for the catch! I went through this too quickly